safe-room.lic

```diff
@@ -72,6 +72,9 @@ class SafeRoom
   def wait_for_healing
     Flags.reset('healthy')
     Flags.reset('moved')
+    
+    lie_down
```

Comment:
I put that there in case the person has multiple severe bleeders. Don't want them tending wounds for a few minutes as they bleed out at the autopath.

We can remove the one in the inner loop, but I though to maintain it since the the lie down function does a check to see if they are standing before issuing the command.

Let me know how that sounds.